shared/actions/config/index.tsx

```diff
@@ -394,172 +390,172 @@ const onShowPermissionsPrompt = (
 
 const onAndroidShare = (state: Container.TypedState) => {
   // already loaded, so just go now
-  if (routeToInitialScreenOnce && state.config.startupDetailsLoaded) {
+  if (/*routeToInitialScreenOnce && */ state.config.startupDetailsLoaded) {
     return RouteTreeGen.createNavigateAppend({path: ['incomingShareNew']})
   }
   return false
 }
 
-let routeToInitialScreenOnce = false
```

Comment:
this has been replaced with the nav's built in handling of incoming Links and goes into a custom hook and out of saga land